docs/feature-gates.md

```diff
@@ -55,6 +56,13 @@ manifest provided as part of releases enables this feature by default. If you
 edit the manifest, make sure you do not disable it, as it is needed for correct
 NetworkPolicy implementation for Pod-to-Service traffic.
 
+### EndpointSlice
+
+`EndpointSlice` enable Service EndpointSlice support in Antrea proxy. The
+EndpointSlice API was introduced in Kubernetes 1.16 (alpha) and it is enabled
+by default in Kubernetes 1.17 (beta). This flag will take no effect if AntreaProxy
+is not enabled.
```

Comment:
Please document the behavior if the EndpointSlice API is not enabled

Maybe ti would be good to add a link to https://kubernetes.io/docs/tasks/administer-cluster/enabling-endpointslices/ here as well

Reply:
As you wish.